package.json

```diff
@@ -29,7 +29,7 @@
     "js-yaml": "^4.1.0",
     "markdownlint-cli": "^0.31.1",
     "node-fetch": "^3.2.0",
-    "prettier": "^2.3.2"
+    "prettier": "^2.8.6"
```

Comment:
Why did you update prettier?

Reply:
Not sure, haha. I will revert it back.

Reply:
![image](https://user-images.githubusercontent.com/110294810/227043150-23a0d7e4-0e3b-4cd9-af74-f10d10f85596.png)

Reply:
Hello, @HerbertHomolka1  Well  
1- you shouldn't change any files except students-bio and add your md file and image.
2- if you have any unknown words only in your md file you can them to .cspell.json.
3-  you should only add your changes, commit them and push them.
4- You should never change other files , especially packages.json